pkg/servicemesh/federation/status/manager_test.go

```diff
@@ -734,10 +731,10 @@ func TestStatusManager(t *testing.T) {
 				close(stopChan)
 				t.Fatalf("timed out waiting for leader election")
 			}
-			for !rm.ExportsInformer().Informer().HasSynced() ||
-				!rm.ImportsInformer().Informer().HasSynced() ||
-				!rm.PeerInformer().Informer().HasSynced() {
+
+			for !rm.HasSynced() {
```

Comment:
Can this cause an infinite loop or consume too much cpu for a certain amount of time?

Maybe surround it with retry.UntilSuccess with some timeout and deley between retries?

Reply:
Yeah, you're right. This tight loop was here before this PR, but I should have fixed it.